### PR TITLE
Fixed config file being overriden when it's blocked by another application

### PR DESCRIPTION
**Pull request by drfrag666**

### CODE REVIEW

File: src/common/utility/configfile.cpp [hunk 1]
```diff
@@ -601,10 +604,13 @@ void FConfigFile::LoadConfigFile ()
 	FileReader file;
 	bool succ;
 
-	FileExisted = false;
+	if (!std::filesystem::exists(PathName.GetChars()))
+		FileExisted = false;
```
Comment (madame-rachelle) [inline src/common/utility/configfile.cpp:608]: You changed a guaranteed initialization here to a conditional one. I don't like this. Is this defined elsewhere?

**Review by madame-rachelle — CHANGES_REQUESTED** [review]

File: src/common/utility/configfile.cpp (hunk 1 above)
Comment (drfrag666) [reply src/common/utility/configfile.cpp:608]: Yes, it's set to true in the constructor. It doesn't compile on macOS and gcc 7.

**Comment by coelckers** [comment]
std::filesystem is not available on macOS 10.14 so we cannot use it.

**Comment by drfrag666** [comment]
_stat really didn't work well (the wide version is required on windows) so i've left exists for windows only, probably it's not worth messing with GetFileAttributesW. It's a minor issue anyway.

**Pull request closed without merging**